tests/cli/test_cli_regression_mol+mol.py

```diff
@@ -131,7 +131,7 @@ def test_train_output_structure(monkeypatch, data_path, tmp_path):
         m.setattr("sys.argv", args)
         main()
 
-    assert (tmp_path / "model_0" / "model.pt").exists()
+    assert (tmp_path / "model_0" / "best.pt").exists()
```

Comment:
Additionally, should we add `assert (tmp_path / "model_0" / "last.pt").exists()` in tests? Although, probably no one needs the last file though.

Reply:
I will just remove `last.pt`